### PR TITLE
add consent banner to psammead

### LINKED ISSUE #1934: Move ConsentBanner component from simorgh to psammead
**Is your feature request related to a problem? Please describe.**
The ConsentBanner component should not be in this repo, it should be in bbc/psammead

**Describe the solution you'd like**
https://www.youtube.com/watch?v=hdcTmpvDO0I

**Describe alternatives you've considered**
None

**Testing notes**
[Tester to complete]

Dev insight: It's not currently used, so minimal.

**Additional context**
n/a

### LINKED ISSUE #1934: Timestamp Container story buttons not working
**Describe the bug**
When using the Timestamp Container story (https://bbc.github.io/psammead/?path=/story/containers-timestampcontainer--default), the buttons to change the date/time do not currently work.

**To Reproduce**
1. Visit https://bbc.github.io/psammead/?path=/story/containers-timestampcontainer--default (or run locally)
2. Click on button `20 days ago` (or any other button)
3. The date displayed doesn't change.

And:
1. Visit https://bbc.github.io/psammead/?path=/story/containers-timestampcontainer--default (or run locally)
2. Set the `isRelative` checkbox to true
2. Click on button `2 hours ago` (or any other button)
3. The date displayed displays `x minutes ago` instead of `2 hours ago` (`x` being the amount of time since the page has loaded).

**Expected behaviour**
The buttons should update the time / date.

**Screenshots**
(Taken 29th of Aug 2019)
screenshot-29th-aug

**Desktop:**
- OS: macOS
- Browser: Chrome, Safari

**Testing notes**
- [ ] This bug fix is expected to need manual testing.

### PR DESCRIPTION
**Pull request by thekp**
Resolves https://github.com/bbc/simorgh/issues/1934
accessibility follow up issue:  https://github.com/bbc/psammead/issues/688

**Overall change:** _Pull consent banner from Simorgh to Psammead._

**Code changes:**

- _Create component (psammead-consent-banner) and update simorgh specific imports._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [N/A] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: packages/components/psammead-consent-banner/README.md [hunk 1]
```diff
@@ -12,14 +12,14 @@ The `psammead-consent-banner` component is a styled `div` that encapsulates info
...
 
-| Argument | Type    | Required | Default | Example                                                                                     |
-| -------- | ------- | -------- | ------- | ------------------------------------------------------------------------------------------- |
-| title    | string  | Yes      | N/A     | `We've updated our Privacy and Cookies Policy`                                              |
-| text     | element | Yes      | N/A     | `<p> This is some text</p>`                                                                 |
-| accept   | element | Yes      | N/A     | `<button type="button">Accept</button>`                                                     |
-| reject   | element | Yes      | N/A     | `<a href="https://www.bbc.co.uk/usingthebbc/your-data-matters">Find out what's changed</a>` |
-| id       | string  | No       | Null    | `ConsentBanner`                                                                             |
-| hidden   | bool    | No       | Null    | `false`                                                                                     |
+| Argument | Type | Required | Default | Example |
```
Comment (DenisHdz) [inline packages/components/psammead-consent-banner/README.md:16]: Add this exception `<!-- prettier-ignore -->` to solve this issues :)

Comment (thekp) [reply packages/components/psammead-consent-banner/README.md:16]: thanks, i had to disable my prettier 😅but i'll add that now 👍

File: packages/components/psammead-consent-banner/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Psammead Consent Banner Changelog
+
+<!-- prettier-ignore -->
+| Version | Description |
+|---------|-------------|
+| 0.1.0 | [PR#667](https://github.com/bbc/psammead/pull/667) Create initial package, pulled in from [Simorgh](https://github.com/BBC-News/simorgh). |
```
Comment (dr3) [inline packages/components/psammead-consent-banner/CHANGELOG.md]: Soz, can you start from `1.0.0`, save is bumping it in a few days

File: packages/components/psammead-consent-banner/README.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+const Text = (
+  <ConsentBannerText>
+    This is some text with <a href="https://www.bbc.com/news">a link</a> inside
+    the consent banner. We have made some important changes to our Privacy and
+    Cookie Policy.
+  </ConsentBannerText>
+);
+
+const props = {
```
Comment (dr3) [inline packages/components/psammead-consent-banner/README.md:54]: Can this example call `ConsentBanner` please, feels incomplete

File: packages/components/psammead-consent-banner/README.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+`npm install @bbc/psammead-consent-banner`
+
+## Props
+
+<!-- prettier-ignore -->
+| Argument | Type | Required | Default | Example |
+| -------- | ---- | -------- | ------- | --------|
+| title | string | Yes | N/A | `We've updated our Privacy and Cookies Policy` |
+| text | element | Yes | N/A | `<p> This is some text</p>` |
```
Comment (dr3) [inline packages/components/psammead-consent-banner/README.md]:
```suggestion
| text | element | Yes | N/A | `<ConsentBannerText> This is some text</ConsentBannerText>` |
```

**Review by dr3 — APPROVED** [review]

**Comment by thekp** [comment]
This PR is currently missing accessibility documentation, as there has been no accessibility testing done for it previously. I briefly checked the ConsentBanner with Natalie and Catharine. And we concluded that the banner is already live and the accessibility can be done a part of another issue for this component

**Comment by pjlee11** [comment]
@thekp could you create and link the follow up issue for accessibility testing the ConsentBanner please

**Comment by thekp** [comment]
accessibility follow up issue: https://github.com/bbc/psammead/issues/688

**Review by pjlee11 — APPROVED** [review]

**Comment by thekp** [comment]
I just noticed this issue: https://github.com/bbc/psammead/issues/673

so i am going to bump gel-foundations here as well.

File: packages/components/psammead-consent-banner/src/index.jsx [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+import { string, element, bool } from 'prop-types';
+import styled from 'styled-components';
+import {
+  C_CONSENT_BACKGROUND,
+  C_CONSENT_ACTION,
+  C_CONSENT_CONTENT,
+  C_WHITE,
+} from '@bbc/psammead-styles/colours';
+import {
+  GEL_GREAT_PRIMER,
```
Comment (dr3) [inline packages/components/psammead-consent-banner/src/index.jsx]: Hey, soz just spotted this. Can we update these to use the new functions, and receive a script prop please? An example can be found here https://github.com/bbc/psammead/blob/latest/packages/components/psammead-story-promo/src/index.jsx#L18

**Review by dr3 — CHANGES_REQUESTED** [review]

**Review by dr3 — APPROVED** [review]

**Review by pjlee11 — APPROVED** [review]
Noice

**Comment by PriyaKR** [comment]
I am seeing error in the console `KnobManager.js:160 Uncaught TypeError: Cannot read property 'emit' of undefined` when i try to click `Find out what's changed` link and come back again to storybook.

I have verified on psammead latest and simorgh latest to see if this is an existing issue.I can't reproduce there.

**Comment by thekp** [comment]
@PriyaKR I found that this is a running issue on Psammead components that do not use `knob` on storybook. 
there is an issue raised to add `knob` to this component here: https://github.com/bbc/psammead/issues/700

and i have raised another issue to fix the overall issue: https://github.com/bbc/psammead/issues/702

**Comment by PriyaKR** [comment]
Looks good to me.

**Pull request merged**